src/OSPSuite.Core/Domain/ObjectPath.cs

```diff
@@ -211,20 +205,35 @@ IEnumerator IEnumerable.GetEnumerator()
       }
 
       /// <summary>
-      ///    Add one entry at the front of the path
+      ///    Add path entries to the end of the path
       /// </summary>
-      /// <param name="pathEntry">path entry to add</param>
-      public virtual void AddAtFront(string pathEntry)
+      public virtual void Add(string pathToAdd)
+      {
+         var pathElements = splitToElements(pathToAdd);
+         pathElements.Each(x => _pathEntries.Add(x));
+      }
+
+      /// <summary>
+      ///    Add path entries to the front of the path
+      /// </summary>
+      public virtual void AddAtFront(string pathToAdd)
+      {
+         var pathElements = splitToElements(pathToAdd);
+         pathElements.Reverse().Each(x => _pathEntries.Insert(0, x));
+      }
+
+      private static string[] splitToElements(string pathToInsert)
       {
-         _pathEntries.Insert(0, pathEntry);
+         var pathItems = pathToInsert.Split(new[] {PATH_DELIMITER}, StringSplitOptions.None);
```

Comment:
there is a ToPathArray method. . I would use this one

Reply:
hmm, some tests break when using that method